testem.js

```diff
@@ -0,0 +1,14 @@
+var Reporter = require('./reporter')
+
+module.exports = {
+  framework: 'mocha',
+  'test_page": 'tests/index.html?hidepassed&coverage',
+  'disable_watching': true,
+  'launch_in_ci': [
```

Comment:
I converted from a `.json` file to a `.js` file. Not sure why Github is highlighting things in red.